CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # did-veres-one ChangeLog
 
+## 4.0.0 - 2019-01-08
+
+### Changed
+- **BREAKING** Using the new flatter `Ed25519VerificationKey2018` type suites.
```

Comment:
Perhaps reword this to avoid confusion:

`Replace "cryptographic suites" with flat verification methods like `Ed25519VerificationKey2018`.

Reply:
fixed.